packages/typespec-ts/test/azureModularIntegration/versioningRemoved.spec.ts

```diff
@@ -47,11 +47,14 @@ describe("VersioningRemoved Rest Client", () => {
     assert.strictEqual(result.enumProp, "enumMemberV1");
   });
 
-  it("versioning removed test modelV33_V2preview", async () => {
+  // Not supported yet
+  // One column is removed in old version but added in latest version
+  // The only way is to cast to any
+  it.skip("versioning removed test modelV33_V2preview", async () => {
```

Comment:
Skipped this case due to we don't support this formally. We couldn't bypass this because spector has more strict check: https://github.com/microsoft/typespec/pull/6565.

Reply:
@v-jiaodi Could you have another pr to re-write this case? We need to build different vesion for different api-version not the same version to share them.

Reply:
issue tracked: https://github.com/Azure/autorest.typescript/issues/3149